appsmithorg/appsmith
# fix: show more modal of a truncated text widget is flickering

## Description

It fixes the bug that when a truncated text widget is added in a modal, the "show more" popover used to flicker on hover.

Fixes #16387


## Type of change

- Bug fix (non-breaking change which fixes an issue)


## How Has This Been Tested?

- Manual

- Steps for Testing

1. Create a Modal
2. Add a text widget inside it
3. Fill the text widget with long text
4. Set the overflow property of the Text widget to "Truncate"
5. Click on the triple dot icon on the Text widget
6. When viewing the 'Show More' popover on top of the Modal, it should not flicker while hovering


## Checklist:
### Dev activity


### QA activity:
